quickwit/quickwit-cli/src/service.rs

```diff
@@ -81,12 +81,12 @@ impl RunCliCommand {
         quickwit_telemetry::send_telemetry_event(telemetry_event).await;
         // TODO move in serve quickwit?
         start_actor_runtimes(&config.enabled_services)?;
-        let _ = serve_quickwit(config, async move {
+        let shutdown_signal = Box::pin(async move {
             signal::ctrl_c()
                 .await
-                .expect("Failure listening for CTRL+C signal")
-        })
-        .await?;
+                .expect("Registering a signal handler for SIGINT should not fail.");
```

Comment:
@imotov

https://doc.rust-lang.org/std/result/enum.Result.html#recommended-message-style
https://doc.rust-lang.org/std/error/index.html#common-message-styles

Reply:
@guilload good to know! Thanks!